Update sql-data-warehouse-tables-partition.md

lucaferrari77 opened the pull request:
Hi 
I did some minor change to T-SQL code and to some statement.

lucaferrari77 committed:
Update sql-data-warehouse-tables-partition.md

Hi 
I did some minor change to T-SQL code and to some statement.

Court72 commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
```diff
@@ -1,6 +1,6 @@
 ---
-title: Partitioning tables in dedicated SQL pool 
-description: Recommendations and examples for using table partitions in dedicated SQL pool.
+title: Partitioning tables in Dedicated SQL pool 
```
```suggestion
title: Partitioning tables in dedicated SQL pool 
```

Court72 committed:
Apply suggestions from PR review

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
```diff
@@ -1,6 +1,6 @@
 ---
 title: Partitioning tables in dedicated SQL pool 
-description: Recommendations and examples for using table partitions in dedicated SQL pool.
+description: Recommendations and examples for using table partitions in Dedicated SQL pool.
```
```suggestion
description: Recommendations and examples for using table partitions in dedicated SQL pool.
```
In this context, dedicated is not capitalized.

WilliamDAssafMSFT replied:
@lucaferrari77 These and other in-line suggestions need to be accepted into your branch before merging. Hit "commit"  beneath each of my suggestions and you'll see my suggestion applied as a commit. You can accept multiple commits as a batch as well! This method of collaborating in GitHub is a little new, let me know if you have any questions.

lucaferrari77 replied:
@WilliamDAssafMSFT I cannot find the "commit" button for these in-line suggestion
Would you please help me ?

WilliamDAssafMSFT replied:
Unresolve the conversations, then you should see buttons like this for each of our in-line code suggestions:
![image](https://user-images.githubusercontent.com/74387232/168622777-0e430dc8-481e-497a-9dd1-56f524383555.png)

lucaferrari77 replied:
@WilliamDAssafMSFT I did unresolve but Commit is not working.
It says "Outdated suggestions cannot be applied"
Would you please help on this ?
![image](https://user-images.githubusercontent.com/90248940/168760776-71dbab9b-4537-4f23-b76b-260583c3df1f.png)

WilliamDAssafMSFT replied:
In that particular case, looks like you figured it out. This commit https://github.com/MicrosoftDocs/azure-docs/pull/92544/commits/fc1c924b5481b80986e9d58cf39f1716421b6987 is in place.

Unresolve and commit this co-authored edit too: https://github.com/MicrosoftDocs/azure-docs/pull/92544/files#r870817696

Then we should be good to move this forward! Thanks again for taking the time to improve this article!

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
```diff
@@ -87,7 +86,7 @@ To migrate SQL Server partition definitions to dedicated SQL pool simply:
 - Add the [partition function](/sql/t-sql/statements/create-partition-function-transact-sql?toc=/azure/synapse-analytics/sql-data-warehouse/toc.json&bc=/azure/synapse-analytics/sql-data-warehouse/breadcrumb/toc.json&view=azure-sqldw-latest&preserve-view=true) definition to your CREATE TABLE.
 
-If you are migrating a partitioned table from a SQL Server instance, the following SQL can help you to figure out the number of rows that in each partition. Keep in mind that if the same partitioning granularity is used in dedicated SQL pool, the number of rows per partition decreases by a factor of 60.  
+If you are migrating a partitioned table from a SQL Server instance, the following SQL can help you to figure out the number of rows that resides in each partition. Keep in mind that if the same partitioning granularity is used in dedicated SQL pool, the number of rows per partition decreases by a factor of 60.  
```
```suggestion
If you are migrating a partitioned table from a SQL Server instance, the following SQL can help you to figure out the number of rows in each partition. Keep in mind that if the same partitioning granularity is used in dedicated SQL pool, the number of rows per partition decreases by a factor of 60.  
```

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
````diff
@@ -234,16 +230,18 @@ AS
 ```
 
-Once you have completed the movement of the data, it is a good idea to refresh the statistics on the target table. Updating statistics ensures the statistics accurately reflect the new distribution of the data in their respective partitions.
+Once you have completed the movement of the data, it is a good idea to refresh the statistics on the target table and to drop unnecessary tables. Updating statistics ensures the statistics accurately reflect the new distribution of the data in their respective partitions.
````
```suggestion
Once you have completed the movement of the data, it is a good idea to refresh the statistics on the target table. Updating statistics ensures the statistics accurately reflect the new distribution of the data in their respective partitions.
```
Not every ETL involves dropping partition switch tables. Would be confusing guidance for some customers. Will add guidance differently.

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
```diff
@@ -234,16 +230,18 @@ AS
 UPDATE STATISTICS [dbo].[FactInternetSales];
+DROP TABLE [dbo].[FactInternetSales_20000101]
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
```
```suggestion
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
```

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
````diff
@@ -234,16 +230,18 @@ AS
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
 ```
 
````
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`.  Alternatively, you may want to keep empty tables for regular, automated partition switches.

WilliamDAssafMSFT reviewed (changes requested):
Hi @lucaferrari77! Thanks for contacting Microsoft with feedback about our product documentation! Added some suggestions and changes to your proposals. Please review and accept. When comments are resolved, let me know and we'll review moving this forward.

lucaferrari77 committed:
Update articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

Co-authored-by: William Assaf MSFT <wiassaf@microsoft.com>

lucaferrari77 committed:
Update articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

Co-authored-by: William Assaf MSFT <wiassaf@microsoft.com>

lucaferrari77 committed:
Update articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

Co-authored-by: William Assaf MSFT <wiassaf@microsoft.com>

lucaferrari77 reviewed (commented):
It seems "Approve" option button is not enabled.
Is this correct ?

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
```diff
@@ -244,6 +239,9 @@ Once you have completed the movement of the data, it is a good idea to refresh t
 UPDATE STATISTICS [dbo].[FactInternetSales];
+DROP TABLE [dbo].[FactInternetSales_20000101]
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
```
```suggestion
```

WilliamDAssafMSFT commented on articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md:
````diff
@@ -244,6 +239,9 @@ Once you have completed the movement of the data, it is a good idea to refresh t
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
 ```
 
````
```suggestion
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
```

WilliamDAssafMSFT replied:
@lucaferrari77 this PR isn't ready, I'm not sure what happened but please unresolve this comment and commit the suggestion "Finally, in the case..."

lucaferrari77 replied:
@WilliamDAssafMSFT, I cannot "Unresolve" this. It only shows "Resolve conversation"
Not sure how to proceed.
![image](https://user-images.githubusercontent.com/90248940/172113058-c739529b-239f-4610-a299-06bd6a97790e.png)

WilliamDAssafMSFT replied:
Just above that - I can see the two buttons in your screenshot - do you have an option to **Commit Suggestion**?

WilliamDAssafMSFT replied:
If not, @Lucaferrari77 can you please add the following text as a new commit?

```
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
```

lucaferrari77 committed:
Update articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

Co-authored-by: William Assaf MSFT <wiassaf@microsoft.com>

lucaferrari77 committed:
New commit

I added this:
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`. Alternatively, you may want to keep empty tables for regular, automated partition switches.
